lib/src/snippet/kotlin/com/lemonappdev/konsist/LibrarySnippets.kt

```diff
@@ -0,0 +1,98 @@
+package com.lemonappdev.konsist
+
+import com.lemonappdev.konsist.api.Konsist
+import com.lemonappdev.konsist.api.declaration.KoDeclaration
+import com.lemonappdev.konsist.api.ext.sequence.withName
+import com.lemonappdev.konsist.api.ext.sequence.withoutAnnotationsOf
+import com.lemonappdev.konsist.core.verify.assert
+import com.lemonappdev.konsist.core.verify.assertNot
+import org.junit.jupiter.api.RepeatedTest
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestFactory
+import org.junit.jupiter.api.TestTemplate
+import org.junit.jupiter.params.ParameterizedTest
+
+class LibrarySnippets {
+    fun `every api declaration has KDoc`() {
+        Konsist.scopeFromPackage("..api..")
+            .declarations(includeNested = true)
+            .assert { it.hasKDoc() }
+    }
+
+    fun `every api declaration has complete KDoc`() {
+        Konsist.scopeFromPackage("..api..")
+            .declarations(includeNested = true)
+            .assert { it.hasValidKDoc() }
+    }
+
+    fun `test classes should reside in the same package as tested class`() {
+        Konsist.scopeFromProduction()
+            .classes()
+            .filter { it.hasTest() }
+            .assert {
+                Konsist.scopeFromTest()
+                    .classes()
+                    .withName("${it.name}Test")
+                    .first()
+                    .packagee == it.packagee
+            }
+    }
+
+    fun `test classes should have test subject named sut`() {
+        Konsist.scopeFromTest()
+            .classes()
+            .assert {
+                val type = it.name.removeSuffix("Test")
+                val sut = it
+                    .properties()
+                    .firstOrNull { property -> property.name == "sut" }
+
+                sut != null && sut.type?.name == type
+            }
+    }
+
+    fun `test classes should have all members private besides tests`() {
```

Comment:
Remove we have ticket for this https://lemonappdev.atlassian.net/browse/KON-243

Reply:
This ticket is about different test - `test classes should reside in the same package as tested class`. 

I changed above test to our updated version.